content/get_start/os_cli/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Use with OS App using CLI"
+date = 2022-09-21T14:49:40-07:00
+weight = 22
+chapter = true
+pre = "<b>2. </b>"
++++
+
+### Chapter 2.2
+
+# Use with OpenStudio APP using CLI
```

Comment:
Should update this to Use with OpenStudio SDK CLI - The CLI is from OpenStudio SDK, not from APP.

Refer to the description: http://openstudio.net second paragraph